PYME/LMVis/visCore.py

```diff
@@ -421,6 +422,26 @@ def OnViewInterpTriangles(self,event):
         self.RefreshView()
         #self.CreateFoldPanel()
         self.displayPane.OnPercentileCLim(None)
+
+    def OnLoadExtraDatasource(self, event):
+
+        def ask(parent=None, message='', default_value=''):
+            with wx.TextEntryDialog(parent, message, value=default_value) as dlg:
+                dlg.ShowModal()
+                result = dlg.GetValue()
+            if result == '':
+                return None
+            else:
+                return result
+        
+        filename = wx.FileSelector("Choose a file to open", 
```

Comment:
here we should probably add a wildcard flag with all reasonable extensions for which we can currently load datasources (in other words quite a few)...